src/unix/core.c

```diff
@@ -1215,6 +1223,13 @@ int uv__getpwuid_r(uv_passwd_t* pwd) {
   pwd->shell = pwd->homedir + homedir_size;
   memcpy(pwd->shell, pw.pw_shell, shell_size);
 
+  /* Copy the gecos field */
+  if (gecos_size > 0) {
+    pwd->gecos = pwd->shell + shell_size;
+    memcpy(pwd->gecos, pw.pw_gecos, gecos_size);
+  } else
+    pwd->gecos = "";
```

Comment:
Ok, good point, I didn't realise. Should I just allocate one byte and set it to `\0` or how else do I do this?